publication-event-handlers/src/test/java/no/unit/nva/publication/events/handlers/expandresources/RecoveryEntryTest.java

```diff
@@ -36,4 +44,14 @@ void shouldThrowExceptionWhenCreatingRecoveryEntryFromUnsupportedEntity() {
 
         assertThrows(IllegalStateException.class, () -> RecoveryEntry.fromDataEntryUpdateEvent(dataEntryUpdateEvent));
     }
+
+    @Test
+    void shouldCreateRecoveryEntryForFileEntryFromDataEntryUpdateEvent() {
```

Comment:
always, or when data other that version change?

Reply:
we really like to batchscan, will this generate lots of noise in logs?

Reply:
sounds good :-) easy to adjust later 👍

Reply:
Otherwise we have to store dao version on sqs message itself to compare with the one we are fetching. We can look at it when we get back to it :)